### PR TITLE
Doco updates about prerequisites and how do add new projects.

### PR DESCRIPTION
**Pull request by ramonsmits**
@timbussmann @dvdstelt Please review if the added guidance is clear or if it needs to be extended.

### CODE REVIEW

**Comment by dvdstelt** [comment]
LGTM

File: src/PerformanceTests/readme.md [hunk 1]
```diff
@@ -22,10 +22,18 @@ Table of contents
...
 
-# Prerequisites for running tests
+# Prerequisites
 
-- Making sure that connection string to MS SQL DB is correct
-- Creating empty database manually before first run
+Tools for development and testing:
+
+- Visual Studio 2017 (tested with 15.3.5)
+- [Slow Cheetah for vs2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.SlowCheetah-XMLTransforms)
```
Comment (timbussmann) [inline src/PerformanceTests/readme.md]: is this really required anymore? I see a Microsoft-vs-slow-cheetah nuget package on some projects but that doesn't seem to require the slow cheetah plugin?

File: src/PerformanceTests/readme.md [hunk 2]
```diff
@@ -22,10 +22,18 @@ Table of contents
...
+
+
+Configuration and environment:
+
+- Make sure that MSSQL connection strings are correct
+- A (empty) MSSQL database, this needs to be created manually
```
Comment (timbussmann) [inline src/PerformanceTests/readme.md]: name of the database?

File: src/PerformanceTests/readme.md [hunk 3]
```diff
@@ -79,6 +87,40 @@ You can verify this by opening the corresponding folder in Windows Explorer. The
...
+* Add '.x64' to Assembly name (NServiceBus7 => NServiceBus7.x64)
+* Default namespace : "Host"
+* remove the check at "Prefer 32-bit"
+* copy pre and post build events from another host
+  * Update the executable names in the post build step
+
+Configure Slow Cheetah:
+
+* Add visual studio extension 'slow cheetah'
+* Add nuget package 'slow cheetah'
```
Comment (timbussmann) [inline src/PerformanceTests/readme.md]: according to the v7 branch, the plugin is `Microsoft.VisualStudio.SlowCheetah`?

File: src/PerformanceTests/readme.md (hunk 1 above)
Comment (ramonsmits) [reply src/PerformanceTests/readme.md]: The extension is required in order to view and test the transformations within the v2017 UI but is not needed to build the projects.

File: src/PerformanceTests/readme.md (hunk 2 above)
Comment (ramonsmits) [reply src/PerformanceTests/readme.md]: That depends on your personal connection string.

**Comment by ramonsmits** [comment]
@timbussmann I've addressed your comments, good?

**Pull request merged**